docker/cortx-build/gpgkey-generation.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+if [ "x$1" = "x" ]; then
+        cat /opt/rpm-signing/gpgoptions >>  ~/.rpmmacros
+        sed -i '/passphrase/d' /opt/rpm-signing/genkey-batch
+        sed -i -e 's/$argv 1/$argv 0/g' -e 's/${PASSPHRASE}/ /g' -e '/PASSPHRASE/d' /opt/rpm-signing/rpm-sign.sh
+        /usr/bin/gpg --pinentry-mode loopback --passphrase '' --batch --gen-key /opt/rpm-signing/genkey-batch
```

Comment:
`-pinentry-mode loopback` is required for Rocky Linux only right. So this will run if `OSNAME` is not provided.

Reply:
Removed this pinentry mode